src/core/platforms/platformutilities.cpp

```diff
@@ -291,7 +304,7 @@ ResourceSource *PlatformUtilities::getFile( QQuickItem *parent, const QString &p
   switch ( fileType )
   {
     case AudioFiles:
-      filter = tr( "Audio files (*.mp3 *.aac *.ogg)" );
+      filter = tr( "Audio files (*.mp3 *.aac *.ogg *.m4a *.mp4 *.mov)" );
```

Comment:
Isn't .mov a video file?

Reply:
.mov is a container which can hold video and/or audio. For e.g., Qt was doing audio-only .mov using the QAudioRecorder before setting it to ogg.